components/ee/payment-endpoint/src/config.ts

```diff
@@ -36,11 +37,14 @@ export GITPOD_GITHUB_APP_MKT_NAME=gitpod-draft-development-app
     readonly githubAppAppID: number = process.env.GITPOD_GITHUB_APP_ID ? parseInt(process.env.GITPOD_GITHUB_APP_ID) : 0;
     readonly githubAppWebhookSecret: string = process.env.GITPOD_GITHUB_APP_WEBHOOK_SECRET || "unknown";
     readonly githubAppCertPath: string = process.env.GITPOD_GITHUB_APP_CERT_PATH || "unknown";
+    readonly usageServiceAddr: string = process.env.GITPOD_USAGE_SERVICE_ADDR || "usage.default.svc.cluster.local:9001";
 
-    readonly maxTeamSlotsOnCreation: number = !!process.env.TS_MAX_SLOTS_ON_CREATION ? parseInt(process.env.TS_MAX_SLOTS_ON_CREATION) : 1000;
+    readonly maxTeamSlotsOnCreation: number = !!process.env.TS_MAX_SLOTS_ON_CREATION
+        ? parseInt(process.env.TS_MAX_SLOTS_ON_CREATION)
```

Comment:
Nit: pre-existing unsafe `parseInt` without a radix (can accidentally parse as octal or hex in some edge cases)

```suggestion
        ? parseInt(process.env.TS_MAX_SLOTS_ON_CREATION, 10)
```